client/my-sites/site-settings/date-time-format/start-of-week-option.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+
+export const StartOfWeekOption = ( {
+	disabled,
+	moment,
+	onChange,
+	startOfWeek,
+	translate,
+} ) =>
+<FormFieldset>
+	<FormLabel>
+		{ translate( 'Week starts on' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **88** times:
`translate( 'Week Starts On' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).